easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2rc2-GCC-5.40-2.26.eb

```diff
@@ -0,0 +1,19 @@
+name = 'MVAPICH2'
+version = '2.2rc2'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
```

Comment:
The filename should be fixed to match the specified toolchain, i.e. 

`MVAPICH2-2.2rc2-GCC-5.4.0-2.26.eb`